src/saving/SaveInformation.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+
+/// <summary>
+///   Info embedded in a save file
+/// </summary>
+public class SaveInformation
+{
+    public enum SaveType
+    {
+        /// <summary>
+        ///   Player initiated save
+        /// </summary>
+        Manual,
+
+        /// <summary>
+        ///   Automatic save
+        /// </summary>
+        AutoSave,
+
+        /// <summary>
+        ///   Quick save, separate from manual to make it easier to keep a fixed number of quick saves
+        /// </summary>
+        QuickSave,
+
+        /// <summary>
+        ///   A broken save that cannot be loaded
+        /// </summary>
+        Broken,
```

Comment:
I think this should be named Invalid instead, that sounds more like a more commonly used enum value name rather than broken.